dist/tools/edbg/Makefile

```diff
@@ -4,6 +4,9 @@ PKG_VERSION=935bb4b1efd0e2f8434cf97443e0d95b9e02ec1d
 PKG_LICENSE=BSD-3-Clause
 PKG_BUILDDIR=$(CURDIR)/bin
 
+CC=
```

Comment:
RIOT sets the CC variables when evaluation the toolchain, and for `samr21-xpro` its `gcc-arm-none-eabi` which runs into an error on macOS, see #7200 error message:

```
arm-none-eabi-g++: error: CoreFoundation: No such file or directory
arm-none-eabi-g++: error: IOKit: No such file or directory
arm-none-eabi-g++: error: unrecognized command line option '-framework'
arm-none-eabi-g++: error: unrecognized command line option '-framework'
arm-none-eabi-g++: error: unrecognized command line option '-pthread'
```

As `edbg` is build for flashing, that is **after** RIOT was already build, its safe to simply unset `CC` and let `edbg`s own config+make find the correct compiler again.